lib/rails/generators/mongoid/migration/templates/migration.rb

```diff
@@ -1,5 +1,5 @@
 class <%= migration_class_name %> < Mongoid::Migration
-  def self.up
+<%= "  client :#{client_name}\n\n" if client_name %>  def self.up
```

Comment:
won't this put the `def self.up` on the same line as the client?

Reply:
Nope, there is two `\n` in the generated string. Anyway, I double checked and adding `-` at the end seems to remove the trailing carriage return when the string returned by erb is empty, so the template will be easier to read like this:
```ruby
  <%= "client :#{client_name}\n" if client_name -%>
  def self.up
```

Reply:
Ah ok good point, but yes indeed it looks nicer with the `-%>`. Thanks!